build-system-tests/templates/components/react/next/template-package-11.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "react-17-next-11-ts",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@aws-amplify/ui-react": "^4.5.1",
```

Comment:
Should `ui-react-storage` be in here too?

Reply:
Not here because it's installed later in the workflow